webapp/components/test-search.js

```diff
@@ -159,9 +164,14 @@ const QUERY_SEMANTICS = QUERY_GRAMMAR.createSemantics().addOperation('eval', {
       status: {not: r.sourceString.toUpperCase()},
     };
   },
-  nameFragment: (chars) => {
-    return {pattern: chars.eval().join('')};
+  nameFragment_basic: (basic) => {
+    return {pattern: basic.sourceString};
+  },
+  nameFragment_quoted: (_, chars, __) => {
+    return {pattern: chars.sourceString};
   },
+  backslash: (v) => '\\',
+  quotemark: (v) => '"',
```

Comment:
Why are these two literals special? I don't see operations defined for other literals.